packages/perseus/src/styles/khan-exercise.css

```diff
@@ -1,1492 +1,1496 @@
-table.limit th {
-    font-weight: bold;
-    text-align: center;
-}
-table.limit td {
-    border: 1px solid #aaa;
-}
-table.limit th,
-table.limit td {
-    padding: 5px;
-}
-table.limit th:first-child {
-    text-align: right;
-}
-
-.solutionarea input[type="text"],
-.solutionarea input[type="number"] {
-    width: 80px;
-}
-.solutionarea .short20 input[type="text"],
-.solutionarea .short20 input[type="number"] {
-    width: 20px;
-}
-.solutionarea .short25 input[type="text"],
-.solutionarea .short25 input[type="number"] {
-    width: 25px;
-}
-.solutionarea .short28 input[type="text"],
-.solutionarea .short28 input[type="number"] {
-    width: 28px;
-}
-.solutionarea .short30 input[type="text"],
-.solutionarea .short30 input[type="number"] {
-    width: 30px;
-}
-.solutionarea .short32 input[type="text"],
-.solutionarea .short32 input[type="number"] {
-    width: 32px;
-}
-.solutionarea .short35 input[type="text"],
-.solutionarea .short35 input[type="number"] {
-    width: 35px;
-}
-.solutionarea .short40 input[type="text"],
-.solutionarea .short40 input[type="number"] {
-    width: 40px;
-}
-.solutionarea .short50 input[type="text"],
-.solutionarea .short50 input[type="number"] {
-    width: 50px;
-}
-#readonly {
-    display: none;
-}
-
-.radical .surd {
-    font: 150% Arial;
-    padding: 0 0 0 5px;
-}
-.radical .overline {
-    border-top: 1px solid #000;
-    padding: 6px 1px 0 3px;
-    margin-left: -1px;
-}
-.solutionarea .radical input[type="text"],
-.solutionarea .radical input[type="number"] {
-    width: 40px;
-}
-
-body.debug .graphie {
-    outline: 1px dashed #fdd;
-}
-.graphie svg {
-    position: absolute;
-    top: 0;
-    left: 0;
-}
-
-#scratchpad {
-    display: none;
-    margin: 0 10px;
-    overflow: hidden;
-    padding-bottom: 40px;
-}
-#scratchpad div {
-    box-sizing: border-box;
-    border: 1px solid #ccc;
-    height: 100%;
-    position: absolute;
-    width: 100%;
-    z-index: 1;
-}
-
-#scratchpad-not-available {
-    display: none;
-}
-#extras .report-issue-link {
-    float: right;
-}
-
-#tester-info.info-box {
-    background: #f2e4bf;
-}
-#tester-info .box {
-    border: 1px solid black;
-    padding: 2px 4px;
-    margin-left: 5px;
-}
-#tester-info .group-box {
-    border: 1px solid #aaa;
-    padding: 6px 4px 6px 0px;
-    margin-left: 2px;
-}
-
-#browserwarning {
-    background: white;
-    margin: 0 1px;
-    padding: 18px;
-    font-size: 120%;
-    text-align: center;
-}
-
-#answer_area .answer-buttons input.simple-button,
-#answer_area input.simple-button.full-width {
-    width: 100%;
-}
-#answer_area #answercontent {
-    position: relative;
-    z-index: 2;
-} /* above-scratchpad */
-#answer_area .hint-box {
-    position: relative;
-    z-index: 1;
-}
-
-#problemarea a:link,
+.framework-perseus {
+    .exercise,
+    .vars,
+    #next {
+        display: none;
+    }
+    p.question {
+        font-weight: bold;
+    }
+    var {
+        font-style: normal;
+    }
+
+    .hint.last-hint {
```

Comment:
**thought**: Related to above, could this be `.perseus-hint-renderer:last-child` instead?

Reply:
`perseus-hint-renderer` isn't added in some cases.

Reply:
@jeremywiebe also, concision is the opposite of what we need here. we need to evoke as many classes as possible for it to override global styles in webapp.